www/docs/reactjs/aborting-procedures.md

```diff
@@ -0,0 +1,40 @@
+---
+id: aborting-procedure-calls
+title: Aborting Procedure Calls
+sidebar_label: Aborting Procedure Calls
+slug: /reactjs/aborting-procedure-calls
+---
+
+By default, tRPC does not cancel requests on unmount. If you want to opt into this behaviour, you can provide `abortOnUnmount` in your configuration.
```

Comment:
Does react query have a docs link on what this means? I think TKDodo said something about this a while back

Reply:
Honestly I didn't rewrite this, it's just split out from the other file. Though we need to do a full pass on breaking up the `server/` pages into the integrations as they're very overloaded right now and it's a key cause of users struggling to find the content they want (it's technically in the wrong place!)

Reply:
Looks like it is a bespoke property by the way, it's not a RQ thing, so makes sense to document it here